api/client/ffs_test.go

```diff
@@ -0,0 +1,27 @@
+package client
+
+import (
+	"testing"
+
+	pb "github.com/textileio/powergate/ffs/pb"
+)
```

Comment:
Wanted to start creating tests for the client, but realized all the client tests need to be updated to use whatever docker image method we use in other tests to get the node up and running. Will make and issue and work on that later.

Reply:
Sounds good to get the CLI covered!